src/Components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,70 @@
+import "./Sidebar.css";
+import Item from "../Item/Item";
+import InfoIcon from "@mui/icons-material/Info";
+import LanguageIcon from "@mui/icons-material/Language";
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import { useState } from "react";
+
+const Sidebar = () => {
+  const [isCollapsed, setisCollapsed] = useState(false);
+  const items = [
+    { icon: <LanguageIcon />, itemName: "All Countries" },
+    { icon: <LocationOnIcon />, itemName: "My country" },
+    { icon: <InfoIcon />, itemName: "General Info" },
+  ];
+  const onlyIcons = [
+    { icon: <LanguageIcon /> },
+    { icon: <LocationOnIcon /> },
+    { icon: <InfoIcon /> },
+  ];
+
+  const renderSidebarItems = (items) => {
+    if (items.length) {
+      return items.map((item, index) => {
+        return (
+          <div key={index} className="Sidebar-Container">
+            <Item icon={item.icon} itemName={item.itemName} />
+          </div>
+        );
+      });
+    }
+    return null;
+  };
+  const handleClick = () => {
+    setisCollapsed(true);
+  };
+  return (
+    <div
+      className={`Menu${isCollapsed ? " Menu-collapsed" : " Menu-expanded"}`}
```

Comment:
You may use a space after Menu instead of giving spaces in the string as it can be confusing.